node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -532,22 +533,21 @@ public void updateContainerNodeMetrics() {
         // CPU usage by a container as percentage of total host CPU, cpuPercentageOfHost, is given by dividing used
         // CPU time by the container with CPU time used by the entire system.
         // CPU usage by a container as percentage of total CPU allocated to it is given by dividing the
-        // cpuPercentageOfHost with the ratio of container resources over total host resources. This calculation
-        // assumes that the ratio between container and host resources for disk, memory, and cpu is roughly equal
-        // and therefore only calculates the ratio of container memory against host memory.
+        // cpuPercentageOfHost with the ratio of container minCpuCores by total number of CPU cores.
         double cpuPercentageOfHost = lastCpuMetric.getCpuUsagePercentage(cpuContainerTotalTime, cpuSystemTotalTime);
-        double cpuPercentageOfAllocated = getInverseContainerShareOfHost(nodeSpec) * cpuPercentageOfHost;
+        Optional<Double> cpuPercentageOfAllocated = nodeSpec.minCpuCores.map(containerNumCpuCores ->
```

Comment:
We allocate 10 shares per CPU core, and `node-admin` is allocated exactly 10 shares, so this could actually be easily taken into account by doing a `-1` on `totalNumCpuCores`.